### PR TITLE
Updating with additional AutoGluon context

### PR DESCRIPTION
**Pull request by miroenev**

### CODE REVIEW

**Comment by miroenev** [comment]
Closes #86

File: aws/autogluon/autogluon_airline.ipynb [hunk 1]
```diff
@@ -2,7 +2,78 @@
  "cells": [
```
Comment (zronaghi) [inline aws/autogluon/autogluon_airline.ipynb:2]: <p>Can we update the lifecycle script with RAPIDS 0.19?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (zronaghi) [inline aws/autogluon/autogluon_airline.ipynb:2]: <p>You mentioned this data is also available in us-east region?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Comment by miroenev** [comment]
Closes #112

**Review by zronaghi — APPROVED** [review]

**Pull request merged**